src/hooks/useUploadsManager/useUploadsManager.tsx

```diff
@@ -59,9 +41,8 @@ export const UploadManagerProvider: React.FC = ({ children }) => {
         id_in: uploadsState.filter((item) => item.parentObject.type === 'video').map((item) => item.parentObject.id),
       },
     },
-    { skip: !uploadsState.length }
+    { skip: !uploadsState.length, fetchPolicy: 'network-only' }
```

Comment:
I probably should not add this, but without this policy new videos weren't added to uploads.
Also as a sidenote - it partly fixes #647 - maybe it's worth investigating in the future